tests/helpers/auth-groups.ts

```diff
@@ -0,0 +1,160 @@
+import { Page } from "@playwright/test";
+import { expect } from "../fixtures/lxd-test";
+import { randomNameSuffix } from "./name";
+
+export const randomGroupName = (): string => {
+  return `playwright-group-${randomNameSuffix()}`;
+};
+
+export const visitGroupsPage = async (page: Page) => {
+  await page.goto("/ui/");
+  await page.getByRole("button", { name: "Permissions" }).click();
+  await page.getByRole("link", { name: "Groups", exact: true }).click();
+  await expect(
+    page
+      .getByRole("heading", { name: "Groups Learn more about" })
+      .locator("div"),
+  ).toBeVisible();
+};
+
+export const selectGroupAction = async (
+  page: Page,
+  groupName: string,
+  action:
+    | "Edit group"
+    | "Manage identities"
+    | "Manage permissions"
+    | "Delete group",
+) => {
+  await page
+    .getByRole("row", { name: `Select ${groupName} Name` })
+    .getByLabel("group actions menu")
+    .click();
+  await page.getByRole("button", { name: action }).first().click();
+};
+
+export const renameGroup = async (
+  page: Page,
+  groupName: string,
+  newGroupName?: string,
```

Comment:
```suggestion
  newGroupName: string,
```
This seems more reasonable? we can also drop the condition with if below in the body.